### ISSUE TITLE
json response data larger than 64 kilobytes for anonymous user, wet-boew >= 4.0.44.5 dislikes this.  Please advise how to compile wet-boew - 4.0.43.2 worked, anything >=4.0.44.5 does NOT.   

### ISSUE THREAD

**Issue by joejoseph00** [wet-boew/wet-boew#9456]
I've narrowed down precisely the last working version of wet-boew (4.0.43.2) for this use case any newer version fails to return a proper ajax response.

With 4.0.43.2 the ajax call data returned is valid and as expected.
with the next higher version and all higher versions starting from >=4.0.44.5 the ajax response is empty data or truncated/incorrect.  This breaks our application.

As a workaround, pinning to 4.0.43.1 (or 4.0.43.2) works.  These are the two most recent working versions of wet-boew for this use case.

I'd like to help debug this by compiling and testing the wet-boew project javascript /sass/scss into .min.js or .src.js and .css, I would appreciate assistance in the compiling process/setup.  If a working compiling docker container was published to docker hub and some quick instructions for usage was shared I could assist in debugging this issue and perhaps write a patch and make a merge request.

I'm not sure how to compile/build the wet-boew library , if I knew how to compile it I could go through the various commits between 4.0.43.2 and 4.0.44.5 and see which exact commit causes the bug I'm seeing.  Is there a pre-configured docker image that I could use for compiling/building wet-boew so that I don't have to install a mess of compile dependencies on my machine directly?  I'd prefer to compile inside a docker image.

A hint, the json was larger than 64 kilobytes, this might be now overloading wet-boew. Also, could possibly be related to CSRF restriction changes affecting anonymous users.

Firstly I'd like to say that normally wet-boew should be logging at least something to the console, this time nothing at all was logged.  There is no js exception, no console log.  Is there a debug mode that can be enabled with wet-boew dist js?

Prior to 4.0.44.5, when I use ckeditor in anonymous mode on our web application (Drupal based) advanced functionality like media browsing (intranet site), a media library dialog window loads and we can browse through images, upload new images, select an image to use, filter images and documents via exposed filters.  This involves an ajax call that has > 64kb of data.

But updating to anything >=4.0.44.5 this functionality stops working and there's no javascript exception.  I've gone through every change we have, isolated the problem to upgrading wet-boew from 4.0.43.1 to 4.0.50.1, further investigation shows that 4.0.43.2 works but >=4.0.44.5 does not work.  Due to the lack of a javascript error I do not see a failure point.  The only thing I know is that the regression in behavior is caused by changing versions of wet-boew.  I'm not sure why wet-boew is interfering with our ajax, it should really mind it's own business right?

We've got a request that has 740 characters in the url not including the domain, json data is returned as expected.  However with 4.0.44.5 and the resulting json is not processed as expected and I'm not sure why other than something in 4.0.44.5 version of wet-boew and newer than 4.0.43.2 is causing some sort of communication between the client and the server to not occur or maybe just js error is caught and ignored I'm not sure where things go wrong other than between wet-boew versions.  I've compared the request and response headers with 4.0.43.2 /4.0.43.1 which is working against the same environment with wet-boew 4.0.44.5 which is not working.  It appears there were a LOT of changes between 4.0.43.2 and 4.0.44.5  that are somehow causing an issue but I haven't pinpointed it yet.

**Request URL:**

`/en/media-library?media_library_opener_id=media_library.opener.editor&media_library_allowed_types%5B0%5D=image&media_library_allowed_types%5Baudio_file%5D=audio_file&media_library_allowed_types%5Bdocument%5D=document&media_library_allowed_types%5Binstagram%5D=instagram&media_library_allowed_types%5Bslideshow%5D=slideshow&media_library_allowed_types%5Bslideshow_thumbnav%5D=slideshow_thumbnav&media_library_allowed_types%5Btweet%5D=tweet&media_library_allowed_types%5Bvideo%5D=video&media_library_allowed_types%5Bvideo_file%5D=video_file&media_library_selected_type=image&media_library_remaining=1&media_library_opener_parameters%5Bfilter_format_id%5D=full_html&hash=oY1Ll5_5z4bFAv22qllBmpWetWKr7V-mc9I9x0WpJdw&_wrapper_format=drupal_modal`

**les entêtes de la requête:**

```
POST /en/media-library?media_library_opener_id=media_library.opener.editor&media_library_allowed_types%5B0%5D=image&media_library_allowed_types%5Baudio_file%5D=audio_file&media_library_allowed_types%5Bdocument%5D=document&media_library_allowed_types%5Binstagram%5D=instagram&media_library_allowed_types%5Bslideshow%5D=slideshow&media_library_allowed_types%5Bslideshow_thumbnav%5D=slideshow_thumbnav&media_library_allowed_types%5Btweet%5D=tweet&media_library_allowed_types%5Bvideo%5D=video&media_library_allowed_types%5Bvideo_file%5D=video_file&media_library_selected_type=image&media_library_remaining=1&media_library_opener_parameters%5Bfilter_format_id%5D=full_html&hash=oY1Ll5_5z4bFAv22qllBmpWetWKr7V-mc9I9x0WpJdw&_wrapper_format=drupal_modal HTTP/1.1
Accept: application/json, text/javascript, */*; q=0.01
Accept-Encoding: gzip, deflate, br
Accept-Language: fr-FR,fr;q=0.9,en-US;q=0.8,en;q=0.7
Authorization: Basic YWRtaW46aHVhblBhJCQh
Connection: keep-alive
Content-Length: 1626
Content-Type: application/x-www-form-urlencoded; charset=UTF-8
Cookie: prev_page_blockList=yes; AMCVS_A90F2A0D55423F537F000101%40AdobeOrg=1; Drupal.visitor.autologout_login=1666036044; gpv_pu=undefined; gpv_pt=blank%20page%20title; gpv_pthl=blank%20theme; gpv_pc=blank%20page%20creator; gpv_pqs=blank%20query%20string; gpv_url=AcmeIncorporated-qa.9pro.ca%2Fen%2Fhuman-resources%2Fstaffing%2Finternal-employment-opportunity-request-form; AMCV_A90F2A0D55423F537F000101%40AdobeOrg=1176715910%7CMCIDTS%7C19283%7CMCMID%7C10946178687989162472005790754015099778%7CMCAAMLH-1666726922%7C9%7CMCAAMB-1666726922%7CRKhpRz8krg2tLO6pguXWp5olkAcUniQYPHaMWWgdJ3xzPWQmdj0y%7CMCOPTOUT-1666129322s%7CNONE%7CMCSYNCSOP%7C411-19291%7CMCAID%7CNONE%7CvVersion%7C5.4.0
Host: AcmeIncorporated
Origin: https://AcmeIncorporated
Referer: https://AcmeIncorporated/en/human-resources/staffing/internal-employment-opportunity-request-form?test=2
Sec-Fetch-Dest: empty
Sec-Fetch-Mode: cors
Sec-Fetch-Site: same-origin
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36
X-Requested-With: XMLHttpRequest
sec-ch-ua: "Not;A=Brand";v="99", "Chromium";v="106"
sec-ch-ua-mobile: ?0
sec-ch-ua-platform: "Linux"

```

**Les entêtes de la réponse:**

```
HTTP/1.1 200 OK
Date: Tue, 18 Oct 2022 19:59:05 GMT
Server: Apache
Cache-Control: must-revalidate, no-cache, private
X-UA-Compatible: IE=edge
Content-language: en
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
Permissions-Policy: interest-cohort=()
Expires: Sun, 19 Nov 1978 05:00:00 GMT
X-Generator: Drupal 9 (https://www.drupal.org)
X-Drupal-Ajax-Token: 1
Keep-Alive: timeout=5, max=100
Connection: Keep-Alive
Transfer-Encoding: chunked
Content-Type: application/json
```

**The response is json data.**

```
[
   {
      "command":"settings",
      "settings":{
         "ajaxPageState":{
            "theme":"example_bootstrap",
            "libraries":"agri_admin\/agri-empl-news-submit_css,agri_admin\/agri-empl-submit,agri_admin\/page,blazy\/bio.ajax,bootstrap\/popover,bootstrap\/tooltip,ckeditor\/drupal.ckeditor,ckeditor\/drupal.ckeditor.plugins.drupalimagecaption,clientside_validation_jquery\/cv.jquery.ckeditor,clientside_validation_jquery\/cv.jquery.ife,clientside_validation_jquery\/cv.jquery.validate,core\/drupal.collapse,core\/drupal.date,core\/drupal.dialog.ajax,core\/drupal.tableresponsive,core\/drupal.vertical-tabs,core\/internal.jquery.form,dropzonejs\/integration,dropzonejs\/media_library,dropzonejs\/widget,editor\/drupal.editor.dialog,field_group\/core,field_group\/formatter.details,field_group\/formatter.fieldset,filter\/drupal.filter,example_bootstrap\/global-styling,example_bootstrap\/example-externaladobetmstagingjs,media\/media_embed_ckeditor_theme,media_library\/ui,news_empl\/empl-form,node\/drupal.node,node\/form,paragraphs\/drupal.paragraphs.admin,system\/base,text\/drupal.text,views\/views.ajax,views\/views.module,webform_bootstrap\/webform_bootstrap,wxt_library\/theme-gc-intranet-legacy-dev,wxt_library\/wet-boew-dev"
         },
         "media_library":{
            "selection_remaining":1
         },
         "views":{
            "ajax_path":"\/en\/views\/ajax?media_library_opener_id=media_library.opener.editor\u0026media_library_allowed_types%5B0%5D=image\u0026media_library_allowed_types%5Baudio_file%5D=audio_file\u0026media_library_allowed_types%5Bdocument%5D=document\u0026media_library_allowed_types%5Binstagram%5D=instagram\u0026media_library_allowed_types%5Bslideshow%5D=slideshow\u0026media_library_allowed_types%5Bslideshow_thumbnav%5D=slideshow_thumbnav\u0026media_library_allowed_types%5Btweet%5D=tweet\u0026media_library_allowed_types%5Bvideo%5D=video\u0026media_library_allowed_types%5Bvideo_file%5D=video_file\u0026media_library_selected_type=image\u0026media_library_remaining=1\u0026media_library_opener_parameters%5Bfilter_format_id%5D=full_html\u0026hash=oY1Ll5_5z4bFAv22qllBmpWetWKr7V-mc9I9x0WpJdw\u0026_wrapper_format=drupal_modal",
            "ajaxViews":{
               "views_dom_id:009d6ce558ed1a35b8179b344e6d905431afeeecbc38eff0bb24d06cc376e1ec":{
                  "view_name":"media_library",
                  "view_display_id":"widget",
                  "view_args":"image",
                  "view_path":"\/media-library",
                  "view_base_path":"admin\/content\/media-widget",
                  "view_dom_id":"009d6ce558ed1a35b8179b344e6d905431afeeecbc38eff0bb24d06cc376e1ec",
                  "pager_element":0
               }
            }
         },
         "ajaxTrustedUrl":{
            "\/en\/admin\/content\/media-widget\/image?media_library_opener_id=media_library.opener.editor\u0026media_library_allowed_types%5B0%5D=image\u0026media_library_allowed_types%5Baudio_file%5D=audio_file\u0026media_library_allowed_types%5Bdocument%5D=document\u0026media_library_allowed_types%5Binstagram%5D=instagram\u0026media_library_allowed_types%5Bslideshow%5D=slideshow\u0026media_library_allowed_types%5Bslideshow_thumbnav%5D=slideshow_thumbnav\u0026media_library_allowed_types%5Btweet%5D=tweet\u0026media_library_allowed_types%5Bvideo%5D=video\u0026media_library_allowed_types%5Bvideo_file%5D=video_file\u0026media_library_selected_type=image\u0026media_library_remaining=1\u0026media_library_opener_parameters%5Bfilter_format_id%5D=full_html\u0026hash=oY1Ll5_5z4bFAv22qllBmpWetWKr7V-mc9I9x0WpJdw\u0026_wrapper_format=drupal_modal":true,
            "\/en\/media-library?media_library_opener_id=media_library.opener.editor\u0026media_library_allowed_types%5B0%5D=image\u0026media_library_allowed_types%5Baudio_file%5D=audio_file\u0026media_library_allowed_types%5Bdocument%5D=document\u0026media_library_allowed_types%5Binstagram%5D=instagram\u0026media_library_allowed_types%5Bslideshow%5D=slideshow\u0026media_library_allowed_types%5Bslideshow_thumbnav%5D=slideshow_thumbnav\u0026media_library_allowed_types%5Btweet%5D=tweet\u0026media_library_allowed_types%5Bvideo%5D=video\u0026media_library_allowed_types%5Bvideo_file%5D=video_file\u0026media_library_selected_type=image\u0026media_library_remaining=1\u0026media_library_opener_parameters%5Bfilter_format_id%5D=full_html\u0026hash=oY1Ll5_5z4bFAv22qllBmpWetWKr7V-mc9I9x0WpJdw\u0026_wrapper_format=drupal_modal\u0026ajax_form=1\u0026views_display_id=widget":true,
            "\/en\/admin\/content\/media-widget\/image":true,
            "\/en\/media-library?media_library_opener_id=media_library.opener.editor\u0026media_library_allowed_types%5B0%5D=image\u0026media_library_allowed_types%5Baudio_file%5D=audio_file\u0026media_library_allowed_types%5Bdocument%5D=document\u0026media_library_allowed_types%5Binstagram%5D=instagram\u0026media_library_allowed_types%5Bslideshow%5D=slideshow\u0026media_library_allowed_types%5Bslideshow_thumbnav%5D=slideshow_thumbnav\u0026media_library_allowed_types%5Btweet%5D=tweet\u0026media_library_allowed_types%5Bvideo%5D=video\u0026media_library_allowed_types%5Bvideo_file%5D=video_file\u0026media_library_selected_type=image\u0026media_library_remaining=1\u0026media_library_opener_parameters%5Bfilter_format_id%5D=full_html\u0026hash=oY1Ll5_5z4bFAv22qllBmpWetWKr7V-mc9I9x0WpJdw\u0026_wrapper_format=drupal_modal":true,
            "\/en\/media-library?media_library_opener_id=media_library.opener.editor\u0026media_library_allowed_types%5B0%5D=image\u0026media_library_allowed_types%5Baudio_file%5D=audio_file\u0026media_library_allowed_types%5Bdocument%5D=document\u0026media_library_allowed_types%5Binstagram%5D=instagram\u0026media_library_allowed_types%5Bslideshow%5D=slideshow\u0026media_library_allowed_types%5Bslideshow_thumbnav%5D=slideshow_thumbnav\u0026media_library_allowed_types%5Btweet%5D=tweet\u0026media_library_allowed_types%5Bvideo%5D=video\u0026media_library_allowed_types%5Bvideo_file%5D=video_file\u0026media_library_selected_type=image\u0026media_library_remaining=1\u0026media_library_opener_parameters%5Bfilter_format_id%5D=full_html\u0026hash=oY1Ll5_5z4bFAv22qllBmpWetWKr7V-mc9I9x0WpJdw\u0026_wrapper_format=drupal_modal\u0026ajax_form=1":true
         },
         "ajax":{
            "edit-submit--8jp6Bu_9sVM":{
               "url":"\/en\/media-library?media_library_opener_id=media_library.opener.editor\u0026media_library_allowed_types%5B0%5D=image\u0026media_library_allowed_types%5Baudio_file%5D=audio_file\u0026media_library_allowed_types%5Bdocument%5D=document\u0026media_library_allowed_types%5Binstagram%5D=instagram\u0026media_library_allowed_types%5Bslideshow%5D=slideshow\u0026media_library_allowed_types%5Bslideshow_thumbnav%5D=slideshow_thumbnav\u0026media_library_allowed_types%5Btweet%5D=tweet\u0026media_library_allowed_types%5Bvideo%5D=video\u0026media_library_allowed_types%5Bvideo_file%5D=video_file\u0026media_library_selected_type=image\u0026media_library_remaining=1\u0026media_library_opener_parameters%5Bfilter_format_id%5D=full_html\u0026hash=oY1Ll5_5z4bFAv22qllBmpWetWKr7V-mc9I9x0WpJdw\u0026_wrapper_format=drupal_modal\u0026ajax_form=1\u0026views_display_id=widget",
               "callback":[
                  "Drupal\\media_library\\Plugin\\views\\field\\MediaLibrarySelectForm",
                  "updateWidget"
               ],
               "disable-refocus":true,
               "event":"mousedown",
               "keypress":true,
               "prevent":"click",
               "dialogType":"ajax",
               "submit":{
                  "_triggering_element_name":"op",
                  "_triggering_element_value":"Insert selected"
               }
            },
            "auto_select_handler":{
               "callback":"::updateFormCallback",
               "wrapper":"media-library-wrapper",
               "event":"auto_select_media_library_widget",
               "url":"\/en\/media-library?media_library_opener_id=media_library.opener.editor\u0026media_library_allowed_types%5B0%5D=image\u0026media_library_allowed_types%5Baudio_file%5D=audio_file\u0026media_library_allowed_types%5Bdocument%5D=document\u0026media_library_allowed_types%5Binstagram%5D=instagram\u0026media_library_allowed_types%5Bslideshow%5D=slideshow\u0026media_library_allowed_types%5Bslideshow_thumbnav%5D=slideshow_thumbnav\u0026media_library_allowed_types%5Btweet%5D=tweet\u0026media_library_allowed_types%5Bvideo%5D=video\u0026media_library_allowed_types%5Bvideo_file%5D=video_file\u0026media_library_selected_type=image\u0026media_library_remaining=1\u0026media_library_opener_parameters%5Bfilter_format_id%5D=full_html\u0026hash=oY1Ll5_5z4bFAv22qllBmpWetWKr7V-mc9I9x0WpJdw\u0026_wrapper_format=drupal_modal\u0026ajax_form=1",
               "dialogType":"ajax",
               "submit":{
                  "_triggering_element_name":"auto_select_handler"
               }
            }
         },
         "dropzonejs":{
            "instances":{
               "edit-upload--kHQHD17DmuQ":{
                  "maxFilesize":512,
                  "dictDefaultMessage":"Drop files here to upload them",
                  "acceptedFiles":".png,.gif,.jpg,.jpeg",
                  "maxFiles":1,
                  "timeout":0
               }
            }
         },
         "pluralDelimiter":"\u0003",
         "user":{
            "uid":0,
            "permissionsHash":"cb9def2edfa64bffec618f95b55286404721107a8ac4b77ed3f5543d37363ed2"
         }
      },
      "merge":true
   },
   {
      "command":"add_css",
      "data":"\u003Clink rel=\u0022stylesheet\u0022 media=\u0022all\u0022 href=\u0022\/sites\/default\/files\/css\/css_iqumCyLxTuEZJouCYljykkvrfvuMhwKs09hZRF_goyw.css\u0022 \/\u003E\n"
   },
   {
      "command":"insert",
      "method":"append",
      "selector":"body",
      "data":"\u003Cscript src=\u0022\/sites\/default\/files\/js\/js_WM-yaCF6H2jOylfZ8BW94FaJNRk8l9mkxdvWgTopYP0.js\u0022\u003E\u003C\/script\u003E\n",
      "settings":null
   },
   {
      "command":"openDialog",
      "selector":"#drupal-modal",
      "settings":null,
      "data":"data in this attribute was larger than 64kilobytes and I have truncated it to be able to post this message on github",
      "dialogOptions":{
         "dialogClass":"media-library-widget-modal ui-dialog--narrow",
         "title":"Add or select media",
         "height":"75%",
         "width":"auto",
         "autoResize":"true",
         "modal":true
      }
   }
]

```

**Comment by joejoseph00** [comment]
Would it be possible to assist me in doing a build of wet-boew?  Do you have any dev dockers uploaded on dockerhub preconfigured and ready to compile?

**Comment by joejoseph00** [comment]
This could be related to CSRF session token restriction changes, not sure though because I haven't looked at the commits between 4.0.43.2 and 4.0.44.5

**Comment by joejoseph00** [comment]
> Also you can compare releases at https://github.com/wet-boew/wet-boew/releases
> 
> <img alt="Screen Shot 2022-10-20 at 12 39 35 AM" width="470" src="https://user-images.githubusercontent.com/23484460/196857868-3f8a28ab-732b-4dbe-a215-b90712f4abcd.png">

I've compared all those releases. 4.0.32.2 is the most recent working version of wet-boew for this use case.

Could you please share some instructions on compiling wet-boew?  I'd like to go through commits between 4.0.43.2 and 4.0.44.5 to see which commit causes the failure however I don't have a build environment ready.  Would there be a docker based build environment ready for compiling purposes? or do I have to install a specific version of npm and grump myself?  Do you know where the compiling instructions are located?

**Comment by joejoseph00** [comment]
@kodz1

**Comment by joejoseph00** [comment]
@kodz1

**Comment by joejoseph00** [comment]
Thanks to @nancy-hebert for this, the build and compile instructions are here:

https://wet-boew.github.io/v4.0-ci/docs/start-en.html

I haven't tested these build/compile instructions yet but it's a good starting point.

**Comment by duboisp** [comment]
@joejoseph00 thanks for opening up a request.

After reading it, I do think your issue are originating from the security fix we applied to jQuery v2 through wet-boew. That security fix involved to parse through DOMpurify any DOM manipulation action made by jQuery. Ajax request should be out of scope except if they are somehow added to the DOM with jQuery during the process. There is no error that are throw during the sanitization process and I didn't look if DOMPurify have a such option to activate the sanitization error to be logged in the browser console.

That said, we will need more information about what version of ckeditor that are being use in your Drupal CMS and we will need to know if it rely on jQuery to work.

The easiest solution will be to use a ckeditor version that don't rely and are independent from jQuery.

Feel free to join us during one of our weekly Tuesday office hours, all the information (including the meeting link) are available on our [Office Hours wiki page](https://github.com/wet-boew/wet-boew/wiki/WET-Office-hours,-Heures-de-service-de-la-BOEW).

FYI - Regarding to have a version of wet-boew compiling and locally hosted with docker, I am working on it these days and currently I do have a prototype that work but it is somehow not compatible when embracing WSL2 with docker. I will keep you posted my progression on that.

**Comment by joejoseph00** [comment]
Yes ckeditor v4 and it needs jQuery

Ckeditor v5 is an option for the near future.

**Comment by joejoseph00** [comment]
With that said I've locked on an older wetboew for this internal app in this case

**Comment by joejoseph00** [comment]
When I read the CVE, if I recall the related CVE is a hypothetical and disputed "vulnerability" there is no consensus.

**Comment by duboisp** [comment]
Another solution to fix the ckEditor issue will be to check if you can load another version of jQuery that are not assigned globally to the variable `jQuery` neither `$`  and execute/run ckeditor 4 to use that another version.

Or, there is the possibility to create a custom build of wet-boew and take the customized binary from the "dist" folder to install it in your website. Note: There is no plugin in wet-boew that depend on the jQuery fix, the jQuery fix only modify jQuery at its core and all plugin should continue to work as expected, idem for GCWeb plugin. 
To build wet-boew without the jQuery fix you will need to :
* Install all the required software (NodeJS)  
* run `script/setup`
* Modify the `Gruntfile.coffee`  at [line 375](https://github.com/wet-boew/wet-boew/blob/master/Gruntfile.coffee#L375) to remove the `src/core/dep/jquery-fix.js` file being concatenated
* Build wet-boew by running the command `grunt dist`
* All the binary are available in the "dist" folder

FYI - Currently we have no plan, for WET 4, to undo the jQuery security fix we applied.

**Comment by joejoseph00** [comment]
> Another solution to fix the ckEditor issue will be to check if you can load another version of jQuery that are not assigned globally to the variable `jQuery` neither `$` and execute/run ckeditor 4 to use that another version.
> 
> Or, there is the possibility to create a custom build of wet-boew and take the customized binary from the "dist" folder to install it in your website. Note: There is no plugin in wet-boew that depend on the jQuery fix, the jQuery fix only modify jQuery at its core and all plugin should continue to work as expected, idem for GCWeb plugin. To build wet-boew without the jQuery fix you will need to :
> 
> * Install all the required software (NodeJS)
> * run `script/setup`
> * Modify the `Gruntfile.coffee`  at [line 375](https://github.com/wet-boew/wet-boew/blob/master/Gruntfile.coffee#L375) to remove the `src/core/dep/jquery-fix.js` file being concatenated
> * Build wet-boew by running the command `grunt dist`
> * All the binary are available in the "dist" folder
> 
> FYI - Currently we have no plan, for WET 4, to undo the jQuery security fix we applied.

Thanks for this guidance, for now we're locked on an older version of wetboew as described above and it is working on the older version, it is an internal site so the disputed theoretical CVE is unlikely to affect this internal low risk site.  If we have to down the road upgrade wetboew your steps will help. Currently we're using ckeditor v4, there are plans for ckeditor V5 which I have not yet tested and we are at least 6 months away from starting ckeditor V5 work.

**Comment by joel-osc** [comment]
For the record, this fix appears to be breaking core drupal features such as the ajax reset button on a view filter - we have no choice but to keep wet-boew at an older version until this is resolved. This is quite critical.

**Comment by olstjos** [comment]
Looking through the release notes of https://github.com/wet-boew/wet-boew/releases/tag/v4.0.44.5 I do not see a mention of any CVE.  It was mentioned above that this CVE was disputed so I would like to have a look for myself.

**Comment by joejoseph00** [comment]
This is fixed by upgrading to wet-boew 4.0.74

Merci pour votre travail acharné @duboisp @timarney
